src/lightning/data/streaming/dataset.py

```diff
@@ -157,9 +157,12 @@ def __next__(self) -> Any:
         return data
 
 
-def _try_create_cache_dir() -> Optional[str]:
-    if "LIGHTNING_CLUSTER_ID" not in os.environ or "LIGHTNING_CLOUD_PROJECT_ID" not in os.environ:
+def _try_create_cache_dir(input_dir: str) -> Optional[str]:
+    if not os.getenv("LIGHTNING_CLUSTER_ID") or not os.getenv("LIGHTNING_CLOUD_PROJECT_ID"):
         return None
-    cache_dir = os.path.join("/cache/chunks")
+    import hashlib
```

Comment:
let's avoid the local imports. It's a built-in module

Reply:
done !